chrome/skin/default/zotero/overlay.css

```diff
@@ -617,6 +617,15 @@
 	margin-top: 1px;
 }
 
+#zotero-tb-pq-update {
+	list-style-image: url(chrome://zotero/skin/puzzle-arrow.png);
```

Comment:
I guess that icon was added years ago for the Retrieve Metadata context menu on Windows, but the puzzle icon seems pretty specifically for plugins. (The blue arrow is also pretty hard to see against the green.) How about document-search-result.png for now, and create a ticket to replace it with a better icon?

Digression: I'm sort of wondering why we have icons for the progress queues at all. Do we just not want these windows to disappear behind the main window? Are we not able to keep windows on top? (I forget, but I think that might've been broken in Firefox.)

Reply:
Our initial idea was that it's just more convenient if you can close and open a progress window, but we can rethink if that's actually true from usability perspective. From technical side, I think there're issues on macOS to keep that progress window on the top.

Reply:
Yeah, I don't really see any reason why you'd need to — or would even want to — close and reopen at least any of the existing ones. You can always move them out of the way.

But it would be nicer with always-on-top. The one difference from other windows, and perhaps the reason we added the icons, is that, if you don't know where the progress window went, there's no good way to get it back through the main UI short of triggering the action again on unprocessed items. So until we can do that, I guess we should keep the icons.

Reply:
Sure, if cancelling all operations when closing a window isn't a problem then we can do that.